chainer/training/triggers/interval.py

```diff
@@ -40,9 +42,16 @@ def __call__(self, trainer):
         """
         updater = trainer.updater
         if self.unit == 'epoch':
-            prev = self.count
-            self.count = updater.epoch_detail // self.period
-            return prev != self.count
+            epoch_detail = updater.epoch_detail
+            previous_epoch_detail = updater.previous_epoch_detail
```

Comment:
As `previous_epoch_detail` is a newly-added attribute, some users could create custom updaters that does not have the `previous_epoch_detail` attribute. So, could you consider to replace the right hand side with `getattr(updater, 'previous_epoch_detail', <something>)` so that it works with updaters without `previous_epoch_detail`?